JUnit version 4.12
.E
Time: 0.381
There was 1 failure:
1) test0(org.apache.commons.math.analysis.solvers.BrentSolver_ESTest)
org.apache.commons.math.MathRuntimeException$4: function values at endpoints do not have different signs.  Endpoints: [503.252, 3,137.105], Values: [-2,777.714, -1,598,185.327]
	at org.apache.commons.math.MathRuntimeException.createIllegalArgumentException(MathRuntimeException.java:305)
	at org.apache.commons.math.analysis.solvers.BrentSolver.solve(BrentSolver.java:178)
	at org.apache.commons.math.analysis.solvers.BrentSolver_ESTest.test0(BrentSolver_ESTest.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:750)

FAILURES!!!
Tests run: 1,  Failures: 1

